python/hsfs/core/feature_view_engine.py

```diff
@@ -478,7 +478,7 @@ def compute_training_dataset_statistics(
                         "Provided dataframes should be in dict format "
                         "'split': dataframe"
                     )
-                return self._statistics_engine.register_split_statistics(
+                return self._statistics_engine.compute_split_statistics(
```

Comment:
if it register the statistics I think it should be in the function's name as well

Reply:
I renamed the methods that compute and save statistics.